packages/markdownviewer/src/widget.ts

```diff
@@ -59,12 +63,18 @@ class MarkdownViewer extends Widget {
 
     context.pathChanged.connect(this._onPathChanged, this);
 
-    // Throttle the rendering rate of the widget.
-    this._monitor = new ActivityMonitor({
-      signal: context.model.contentChanged,
-      timeout: RENDER_TIMEOUT
+    this._context.ready.then(() => {
```

Comment:
Similar comments as above about 1) the context ready being equivalent to widget ready and 2) will the promise fire again